src/pages/PasswordReset.jsx

```diff
@@ -0,0 +1,60 @@
+/* eslint-disable prettier/prettier */
```

Comment:
Remove unnecessary comments

Reply:
If I uncomment that line the rest of the code keeps breaking because of prettier

Reply:
Okay